src/data/personoversiktHooks.ts

```diff
@@ -69,15 +69,15 @@ export const usePersonoversiktQuery = () => {
     queryFn: fetchPersonoversikt,
     enabled: !!aktivEnhet,
     staleTime: minutesToMillis(5),
-    onError: (error) => {
```

Comment:
Disse query-callbackene er [deprecated](https://github.com/TanStack/query/discussions/5279) i v5 (finnes fortsatt for mutations).
Alternativ måte å gjøre det på med å definere callbacks på `query.meta`-objektet som brukes der hvor `queryClient` settes opp er beskrevet i denne [her](https://tkdodo.eu/blog/breaking-react-querys-api-on-purpose).